packages/server/test/e2e/1_browserify_babel_es201_spec.coffee

```diff
@@ -1,20 +0,0 @@
-e2e      = require("../support/helpers/e2e")
```

Comment:
Why was this spec removed?

Reply:
Somehow, my git client said it was removed. Maybe I saw something wrong. I'll restore it.